### ISSUE TITLE
modify bash scripts which create configurations to allow firmware tp configurations to be built

### ISSUE THREAD

**Issue by ShyamB97** [DUNE-DAQ/daqconf#258]
@wesketchum to create configurations with firmware TPs for all systems I have copied the configurations directory in my working area (`/nfs/sw/dunedaq/dunedaq-v3.2.1-rc-1-fwtp/confgen_test`) and made some changes, these include:

added `np04_daq_fwtp.json`
added `np04_wib_pulser.json`
added `np04_flxcard_fwtp.json`
modified `recreate_all_configurations.sh` so that the new fwtp and pulser configurations are made.

branches needed to get this working are patch brnahces for flxlibs, wibmod and daqconf, once https://github.com/DUNE-DAQ/daqconf/pull/257 is merged.

**Comment by wesketchum** [comment]
no longer relevant.
